src/wp-login.php

```diff
@@ -314,6 +314,15 @@ function login_footer( $input_id = '' ) {
 	<?php
 	$languages = get_available_languages();
 
+	/**
+	 * Filters the Languages select input activation on the login screen.
+	 *
+	 * @since 5.9.0
+	 *
+	 * @param bool Whether to display the Languages select input on the login screen.
+	 */
+	$display_login_language_dropdown = apply_filters( 'display_login_language_dropdown', true );
+
 	if ( ! empty( $languages ) && ! $interim_login ) {
```

Comment:
Yes, I agree. And a slight tweak could give a teeny tiny performance boost too:

```suggestion
	if (
		! $interim_login &&
		/**
		 * Filters the Languages select input activation on the login screen.
		 *
		 * @since 5.9.0
		 *
		 * @param bool Whether to display the Languages select input on the login screen.
		 */
		apply_filters( 'display_login_language_dropdown', true )
	) {
		$languages = get_available_languages();

		if ( ! empty( $languages ) ) {
```

If not an interim login, then check the filter. 
If the filter is true, then get the languages.
If there are languages, then display the language dropdown.